hedera-node/hedera-app/src/test/java/com/hedera/node/app/handle/AdaptedMonoTransitionRunnerTest.java

```diff
@@ -16,102 +16,77 @@
 
 package com.hedera.node.app.handle;
 
-import static com.hederahashgraph.api.proto.java.HederaFunctionality.ConsensusCreateTopic;
```

Comment:
Looks like Spotless shredded this file.  Might want to recover and comment-out the imports as well.